### ISSUE TITLE
What is this?

### ISSUE THREAD

**Issue by torjusb** [michaelenger/sanntid#3]
What's the plan for this package? Should it exist only as a library to talk with Ruter's API, or should it also be a CLI tool for interacting with it?

Looking at the original [Sanntid.js](https://github.com/mathiasnovas/sanntid/), which basically only does the latter. If we wanted to create a CLI tool I guess we could:
1. build it in as a binary
2. create a separate project for it

If we go with option 2, should this then be moved to a more appropriate name? Maybe `sanntid/api`?

**Comment by michaelenger** [comment]
I would like for this to be a package for getting sanntid info, or perhaps a generic Ruter API package. Then we can build a CLI-tool which uses it at a later date.

Also, I helped make the sanntid.js project into something that works as a CLI-script **and** a module.

**Comment by michaelenger** [comment]
I had a little think about this and I realised that if we make it a generic Ruter API package then we will just be wrapping HTTP requests in a JSON-parser. Which is borderline useless.

After reading this [transcript from Gophercon 2015](https://sourcegraph.com/blog/live/gopherconindia/111854129512) I realized that the crux of the language is to be simple and not implement easy-to-make algorithms for the sake of filling the standard library with convenience functions. To that end I suggest we make this a kickass CLI program for getting sanntid data, perhaps in a format that can be piped into something else (not sure what that would be, but whatevs.

This is just an excuse to experiment with Go anyway, so let's just fuck around.
